src/Transformer.ts

```diff
@@ -63,6 +68,78 @@ export class LuaTransformer {
+            case ts.SyntaxKind.ForStatement:
+            case ts.SyntaxKind.ForOfStatement:
+            case ts.SyntaxKind.ForInStatement:
+            case ts.SyntaxKind.SwitchStatement:
+            case ts.SyntaxKind.BreakStatement:
+            case ts.SyntaxKind.TryStatement:
+            case ts.SyntaxKind.ThrowStatement:
+            case ts.SyntaxKind.ContinueStatement:
+            case ts.SyntaxKind.EmptyStatement:
+                return this.visitStatement(node as ts.Statement);
+            // Expressions
+            case ts.SyntaxKind.BinaryExpression:
+            case ts.SyntaxKind.ConditionalExpression:
+            case ts.SyntaxKind.CallExpression:
+            case ts.SyntaxKind.PropertyAccessExpression:
+            case ts.SyntaxKind.ElementAccessExpression:
+            case ts.SyntaxKind.ParenthesizedExpression:
+            case ts.SyntaxKind.TypeAssertionExpression:
+            case ts.SyntaxKind.AsExpression:
+            case ts.SyntaxKind.TypeOfExpression:
+            case ts.SyntaxKind.SpreadElement:
+            case ts.SyntaxKind.NonNullExpression:
+            case ts.SyntaxKind.ClassExpression:
+            case ts.SyntaxKind.TemplateExpression:
+            case ts.SyntaxKind.PostfixUnaryExpression:
+            case ts.SyntaxKind.PrefixUnaryExpression:
+            case ts.SyntaxKind.ArrayLiteralExpression:
+            case ts.SyntaxKind.ObjectLiteralExpression:
+            case ts.SyntaxKind.DeleteExpression:
+            case ts.SyntaxKind.FunctionExpression:
+            case ts.SyntaxKind.ArrowFunction:
+            case ts.SyntaxKind.NewExpression:
+            // Identifier
+            case ts.SyntaxKind.Identifier:
+            // Literals
+            case ts.SyntaxKind.StringLiteral:
+            case ts.SyntaxKind.NoSubstitutionTemplateLiteral:
+            case ts.SyntaxKind.NumericLiteral:
+            // Keywords
+            case ts.SyntaxKind.TrueKeyword:
+            case ts.SyntaxKind.FalseKeyword:
+            case ts.SyntaxKind.NullKeyword:
+            case ts.SyntaxKind.UndefinedKeyword:
+            case ts.SyntaxKind.ThisKeyword:
+            case ts.SyntaxKind.SuperKeyword:
+                return this.visitExpression(node as ts.Expression);
+            // ComputedPropertyName
+            case ts.SyntaxKind.ComputedPropertyName:
+                return this.visitComputedPropertyName(node as ts.ComputedPropertyName);
+            // Blocks
+            case ts.SyntaxKind.Block:
+                return this.visitBlock(node as ts.Block);
+            case ts.SyntaxKind.ModuleBlock:
+                return this.visitModuleBlock(node as ts.ModuleBlock);
+            // EOF TOKEN
+            case ts.SyntaxKind.EndOfFileToken:
+                return this.visitEndOfFileToken(node as ts.EndOfFileToken);
+            default:
+                throw TSTLErrors.UnsupportedKind("Node", node.kind, node);
+        }
+    }
+    public visitStatement(node: ts.Statement): ts.Statement {
```

Comment:
We should probably consistently add a white-line between methods

Reply:
Added in 3e791a4